UserOnboard/Onboard-UserMain.ps1

```diff
@@ -105,6 +105,17 @@ param (
     
 )
 
+#Script Configuration
```

Comment:
add a better message here:   like #Script Configuration  TO BE FILLED ACCORDING TO YOUR SETUP